frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -165,92 +165,91 @@ if Device:isTouchDevice() then
     }
 end
 
+common_settings.navigation = {
+    text = _("Navigation"),
+}
+common_settings.back_to_exit = {
+    text = _("Back to exit KOReader"),
+    sub_item_table = {
+        {
+            text = _("Prompt"),
+            checked_func = function()
+                local setting = G_reader_settings:readSetting("back_to_exit")
+                return setting == "prompt" or setting == nil
+            end,
+            callback = function()
+                G_reader_settings:saveSetting("back_to_exit", "prompt")
+            end,
+        },
+        {
+            text = _("Always"),
+            checked_func = function()
+                return G_reader_settings:readSetting("back_to_exit")
+                           == "always"
+            end,
+            callback = function()
+                G_reader_settings:saveSetting("back_to_exit", "always")
+            end,
+        },
+        {
+            text = _("Disable"),
+            checked_func = function()
+                return G_reader_settings:readSetting("back_to_exit")
+                           == "disable"
+            end,
+            callback = function()
+                G_reader_settings:saveSetting("back_to_exit", "disable")
+            end,
+        },
+    },
+}
+common_settings.back_in_filemanager = {
+    text = _("Back in file browser"),
+    sub_item_table = {
+        {
+            text = _("Defer to back to exit setting"),
```

Comment:
@poire-z Btw, I'm not too happy about this one but I thought "default" was rather unclear.

Reply:
Mhhh, indeed, I don't understand it :) Why the "setting" word?
I don't remember how it works, but why not just "Exit" ?

Reply:
Because it only exits if the back to exit setting is "exit". ;-)

Reply:
It could also be prompt (the default) or ignore.

Reply:
Oh, ok. I would have understood it better with a capitalized Back as the other item. But you don't like that :) "As in Back to exit", "Same as Back to exit".
No real idea :|

Reply:
How about something like this?

## Navigation menu
<img src=https://user-images.githubusercontent.com/202757/53599910-92814280-3ba8-11e9-823a-4fcabdd77ab7.png width=40%>

## Back in file browser submenu
<img src=https://user-images.githubusercontent.com/202757/53599911-9319d900-3ba8-11e9-83e4-10ea4f451dcf.png width=40%>

Reply:
Clever :)